src/main/java/de/zalando/aruha/nakadi/validation/MetadataValidator.java

```diff
@@ -0,0 +1,42 @@
+package de.zalando.aruha.nakadi.validation;
+
+import org.json.JSONObject;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Optional;
+
+public class MetadataValidator implements EventValidator {
+    private static final String DATETIME_FORMAT_STRING = "yyyy-MM-dd'T'HH:mm:ssXXX";
+    private static final String DATETIME_FORMAT_STRING_SECFRAC = "yyyy-MM-dd'T'HH:mm:ss.SSSXXX";
```

Comment:
I'm not really sure, these two format-strings - do they support all possible cases of the rfc?